sklearn/metrics/_classification.py

```diff
@@ -2425,6 +2425,12 @@ def brier_score_loss(y_true, y_prob, *, sample_weight=None, pos_label=None):
     the greater label unless `y_true` is all 0 or all -1, in which case
     `pos_label` defaults to 1.
 
+    A more generalized form of Brier score is implemented in
+    :func:`multiclass_brier_score_loss` that is applicable to the multi-class
+    case as well. When used for the binary case, `multiclass_brier_score_loss`
+    returns Brier score that is exactly twice of the value returned by this
+    function.
+
```

Comment:
Please recall the latex equation of the docstring here.

Reply:
I copied the equation for `mutliclass_brier_score_loss` as asked. But the docstring for `brier_score_loss` did not include the alternate (i.e. original) formula, so added that equation to the docstring as well.